docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,427 @@
+.. code-block:: none
+
+    [p]editrole name <role> <name>
+
+**Description**
+
+Edits the name of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<name>``: The new role name.
+
+.. _admin-command-editrole-color:
+
+""""""""""""""
+editrole color
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole color <role> <color>
+
+**Description**
+
+Edits the color of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<color>``: The new color to assign. |color-input|
+
+**Examples**
+
+* ``[p]editrole color "My role" #ff0000``
+
+* ``[p]editrole color "My role" dark_blue``
+
+.. _admin-command-announce:
+
+^^^^^^^^
+announce
+^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce <message>
+
+**Description**
+
+Announce your message to all of the servers the bot is in.
+
+The bot will announce the message in the guild's announcements channel
+if set, else she will try the system channel (where the new members are
+welcomed with the Discord announcer). If none of these channels are found,
+the bot will use the highest channel in the list.
+
+**Arguments**
+
+* ``<message>``: The message to send.
+
+.. _admin-command-announce-cancel:
+
+"""""""""""""""
+announce cancel
+"""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce cancel
+
+**Description**
+
+Cancels an active announcement.
+
+.. _admin-command-announceset:
+
+^^^^^^^^^^^
+announceset
+^^^^^^^^^^^
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announceset
+
+**Description**
+
+Manage the settings of announcements sent by the bot owners on servers.
+
+.. _admin-command-announceset-channel:
+
+"""""""""""""""""""
+announceset channel
+"""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announceset channel [channel]
+
+**Description**
+
+Sets the channel where the bot owner announcements will be sent.
+
+**Arguments**
+
+* ``[channel=ctx]``: The channel that will be used for bot announcements.
+  |channel-input| Defaults to where you typed the command.
+
+.. _admin-command-announceset-ignore:
+
+""""""""""""""""""
+announceset ignore
+""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announceset ignore [guild]
+
+**Description**
+
+Enables or disables announcements on the selected guild.
+
+**Arguments**
+
+* ``[guild=ctx]``: The server where the announcements will be enabled/disabled.
+    Defaults to the current server.
+
+.. warning:: You need the appropriate permissions if you're trying to edit a
+    different server's setting than where you are invoking the command.
+
+.. _admin-command-serverlock:
+
+^^^^^^^^^^
+serverlock
+^^^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]serverlock
+
+**Description**
+
+Lock a bot to its current servers only.
+
+This means that, once you enable this, the bot will instantly leave all servers
```

Comment:
Maybe clarify or change the ordering to make it a little less confusing? I read this and when I got to "instantly leave all servers" I thought this description was going to be wrong (then I kept reading). Still a bit confusing though.